spec/providers/atmosphere/local_pdp_spec.rb

```diff
@@ -0,0 +1,88 @@
+require 'rails_helper'
+
+describe Atmosphere::LocalPdp do
+  context '#can_manage?' do
+    let(:user) { build(:user, id: 1) }
+    subject { Atmosphere::LocalPdp.new(user) }
+
+    it 'allows to manage owned appliance types' do
+      at = build(:appliance_type, user_id: user.id)
+
+      expect(subject.can_manage?(at)).to be_truthy
+    end
+
+    it 'allows to manage appliance types where management permissions have been granted' do
+      at = build(:appliance_type)
+      create(:user_appliance_type, user: user, appliance_type: at, role: 'manager')
+
+      expect(subject.can_manage?(at)).to be_truthy
+    end
+
+    it 'does not allow to manage unowned appliance types' do
+      at = build(:appliance_type)
```

Comment:
Similar as above: AT does not exist in db so `where` from `can_manage?` function will always return empty set.